p9/messages.go

```diff
@@ -1999,6 +2000,177 @@ func (r *rusymlink) String() string {
 	return fmt.Sprintf("Rusymlink{%v}", &r.rsymlink)
 }
 
+// lock - acquire or release a POSIX record lock
+// from the linux kernel: req = p9_client_rpc(clnt, P9_TLOCK, "dbdqqds", fid->fid, flock->type, (etc.)
+// byte doubleword quad quad doubleword string
+// This confirms this doc:
+// size[4] Tlock tag[2] fid[4] type[1] flags[4] start[8] length[8] proc_id[4] client_id[s]
+// start, length, and proc_id correspond to the analogous fields passed to Linux fcntl(F_SETLK):
+// size[4] Rlock tag[2] status[1]
+// lock is used to acquire or release a POSIX record lock on fid and has semantics similar to Linux fcntl(F_SETLK).
+// Most servers implement almost none of this stuff -- they use berkeley flock -- i.e., whole file locking.
+// Hence most of these parameters are ignored. A good thing, since this kind of locking over a network
+// makes no sense.
+
+// LockType is lock type for Tlock
+type LockType uint8
+
+// These constants define Lock operations: Read, Write, and Un(lock)
+const (
+	ReadLock  LockType = iota
```

Comment:
can you for each of these say which Linux constant they map to in comments, F_RDLCK, F_WRLCK, F_UNLCK